cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/monitor/RuntimeMonitor.java

```diff
@@ -396,8 +388,8 @@ private void clearStates() {
     }
     // TODO: CDAP-14848 merge transactions back together
```

Comment:
Can the transactions be merged together now?

Reply:
Ah yes, thanks!